src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -128,6 +142,19 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void CheckForUnsupportedArgs(Execution::Context& context)
+    {
+        const auto& unsupportedArgs = context.Get<Execution::Data::Installer>()->UnsupportedArguments;
+
+        for (auto unsupportedArg : unsupportedArgs)
+        {
+            if (context.Args.Contains(GetUnsupportedArgumentType(unsupportedArg)))
+            {
+                context.Reporter.Warn() << Resource::String::UnsupportedArgument << " --"_liv << UnsupportedArgumentToString(unsupportedArg) << std::endl;

```

Comment:
It feels like we should classify the unsupported arguments into two categories: warnings and errors.

For the two currently known values, logs would be warnings (either there is no log or we can't change where it logs).  Install location should be an error though.  If I ask for it to be installed in a specific location, but it won't be, I don't want installation to just proceed.

Reply:
Added a helper method called `ShouldErrorForUnsupportedArgument` that determines whether an error should be shown for the unsupportedArgument. If it returns false, a warning message is shown instead.